src/Sylius/Bundle/ResourceBundle/Controller/ParametersParser.php

```diff
@@ -77,6 +78,16 @@ private function parseRequestValue($parameter, Request $request)
             return $this->parseRequestValueExpression(substr($parameter, 5), $request);
         }
 
+        if (0 === strpos($parameter, '!!')) {
+            $parameter = explode(' ', $parameter);
+
+            $parser = trim($parameter[0], '!').'val';
+
+            Assert::oneOf($parser, ['intval', 'floatval', 'doubleval'], 'Variable can be parsed only to int, float or double.');
```

Comment:
We don't need `doubleval`, it's just an alias for `floatval`.

But we do need `boolval` here.